packages/lib/src/interactions/SelectionTool.tsx

```diff
@@ -31,11 +32,12 @@ function SelectionTool(props: Props) {
     id = 'Selection',
     modifierKey,
     disabled,
+    button = Button.Left,
```

Comment:
I give `Left` as the default value for `SelectionTool` meaning that it is the same for `SelectToZoom`.